test/scripts/install_chromedriver_travis.sh

```diff
@@ -13,11 +14,17 @@ echo "Uninstalling current Google Chrome from `which google-chrome`..."
 sudo apt-get purge google-chrome-stable
 rm ~/.config/google-chrome/ -rf
 
-echo "Installing $CHROME_VERSION..."
-sudo wget --no-verbose -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add -
-echo "deb http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google-chrome.list
-sudo apt-get update -qqy
-sudo apt-get -qqy install ${CHROME_VERSION}
+# echo "Installing $CHROME_VERSION..."
+# sudo wget --no-verbose -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add -
+# echo "deb http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google-chrome.list
+# sudo apt-get update -qqy
+# sudo apt-get -qqy install ${CHROME_VERSION}
+
+# install Chrome 61 due to bug in Chrome 62
+apt-get -y update
```

Comment:
This is a temporary work around right? Is Chrome 62 tagged as stable even though it isn't? What's the bug?

Reply:
Yes this is temporary. It actually might be a problem with chromedriver rather than browser, I'll commit now a change to check that out